src/styles/stories/Paddings/PaddingExample.tsx

```diff
@@ -0,0 +1,50 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+
+import './PaddingExample.scss';
+
+export enum PaddingSizes {
+  EXTRASMALL = 'xs',
+  SMALL = 'sm',
+  MEDIUM = 'md',
+  LARGE = 'lg',
+  EXTRALARGE = 'xl',
+}
+
+export interface PaddingExampleProps
+  extends React.HTMLAttributes<HTMLDivElement> {
+  size: PaddingSizes;
+}
+
+const getSizeValue = (size: PaddingSizes): string => {
+  let text = '';
+  switch (size) {
+    case 'xs':
+      text = '0.125rem';
+      break;
+    case 'sm':
+      text = '0.25rem';
+      break;
+    case 'md':
+      text = '0.375rem';
+      break;
+    case 'lg':
+      text = '0.5rem';
+      break;
+    case 'xl':
+      text = '0.625rem';
+      break;
+    default:
+      text = 'no value found';
+      break;
+  }
+  return `Padding size : ${text}`;
+};
```

Comment:
When the constant text part of moved outside (see [comment below](https://github.com/lienputnina/react-components-for-you/pull/2/files#r998574799)), this can be simplified in to

```suggestion
const getSizeValue = (size: PaddingSizes): string => {
  switch (size) {
    case 'xs':
      return '0.125rem';
    case 'sm':
      return '0.25rem';
    case 'md':
      return '0.375rem';
    case 'lg':
      return '0.5rem';
    case 'xl':
      return '0.625rem';
    default:
      return 'no value found';
  }
};
```